src/schema/community-district.ts

```diff
@@ -17,6 +17,7 @@ export const communityDistrict = pgTable(
   (table) => {
     return {
       pk: primaryKey({ columns: [table.boroughId, table.id] }),
+      liFtGix: index().using("GIST", table.liFt),
```

Comment:
We'll need a migration to actually apply this spatial index

Reply:
(Rebasing off main first, to make sure this branch also has all the existing migrations before creating new ones)

Reply:
Oop, thanks for catching. Updated